add_to_app/README.md

```diff
@@ -54,4 +54,26 @@ functionality, similar to a full Flutter app.
 ## Issues
 
 If you run into an issue with the sample itself, please file an issue
-in the [Flutter samples repo](https://github.com/flutter/samples/issues).
+in the [main Flutter repo](https://github.com/flutter/flutter/issues).
```

Comment:
Is there a reason for changing from using the `flutter/samples` issue tracker. This seems wrong for a `flutter/samples` sample.

Reply:
Good catch, I  truly don't know how that happened, but it's a result of dealing with merge conflicts.